### PR TITLE
修复启用机器人后，ql bot不能正确安装依赖的问题

### PR DESCRIPTION
**Pull request by fzls**
# TLDR
判断对应库是否已安装的test语句有问题，导致始终不会进入then的分支中，继而导致bot的依赖不会正确安装
因为不熟shell语法，所以直接采用简单的办法，去除判断，直接pip install就可以了，已安装的依赖下次运行也不会再执行的，所以没有必要去判断是否安装

# 改动前表现
```log
root@53444edbfe24:/ql $ ql bot
WARNING: Package(s) not found: qrcode
WARNING: Package(s) not found: Telethon
WARNING: Package(s) not found: Pillow
WARNING: Package(s) not found: python-socks
WARNING: Package(s) not found: async_timeout
## 开始执行... 2021-11-13 03:57:28


1、安装bot依赖...

fetch https://mirrors.tuna.tsinghua.edu.cn/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.tuna.tsinghua.edu.cn/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
OK: 313 MiB in 112 packages

bot依赖安装成功...

2、下载bot所需文件...


bot文件下载成功...

3、安装python3依赖...

Writing to /root/.config/pip/pip.conf

python3依赖安装成功...

4、启动bot程序...

bot启动成功...


## 执行结束... 2021-11-13 03:57:31  耗时 3 秒
```

# 改动后表现
root@53444edbfe24:/ql $ ql bot
## 开始执行... 2021-11-13 03:58:43

1、安装bot依赖...

fetch https://mirrors.tuna.tsinghua.edu.cn/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.tuna.tsinghua.edu.cn/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
OK: 332 MiB in 116 packages

bot依赖安装成功...

2、下载bot所需文件...

bot文件下载成功...

3、安装python3依赖...

Writing to /root/.config/pip/pip.conf
Looking in indexes: https://pypi.tuna.tsinghua.edu.cn/simple
Collecting qrcode==6.1
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/42/87/4a3a77e59ab7493d64da1f69bf1c2e899a4cf81e51b2baa855e8cc8115be/qrcode-6.1-py2.py3-none-any.whl (31 kB)
Collecting Telethon==1.21.1
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/01/b1/5e427cf0fecfbb9a5b69a9317610960995c853302fbc0379e053a9d479f0/Telethon-1.21.1-py3-none-any.whl (515 kB)
Requirement already satisfied: requests==2.25.1 in /usr/lib/python3.9/site-packages (from -r requirements.txt (line 3)) (2.25.1)
Collecting Pillow==8.1.2
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/60/f0/dd2eb7911f948bf529f58f0c7931f6f6466f711bd6f1d81a69dc4edd4e2a/Pillow-8.1.2.tar.gz (45.2 MB)
Collecting python-socks==1.2.4
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/e4/d0/6d56fcaca0f44cbcbc8ab3169a17cd90696382a812506fc7041323567d2b/python_socks-1.2.4-py3-none-any.whl (35 kB)
Collecting async_timeout==3.0.1
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/e1/1e/5a4441be21b0726c4464f3f23c8b19628372f606755a9d2e46c187e65ec4/async_timeout-3.0.1-py3-none-any.whl (8.2 kB)
Collecting prettytable>=2.1.0
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/de/56/554603797416219cae8fd3eae33e5e2d58a7fc73419129b62fa419a67856/prettytable-2.4.0-py3-none-any.whl (24 kB)
Requirement already satisfied: six in /usr/lib/python3.9/site-packages (from qrcode==6.1->-r requirements.txt (line 1)) (1.15.0)
Requirement already satisfied: chardet<5,>=3.0.2 in /usr/lib/python3.9/site-packages (from requests==2.25.1->-r requirements.txt (line 3)) (4.0.0)
Requirement already satisfied: idna<3.3,>=2.5 in /usr/lib/python3.9/site-packages (from requests==2.25.1->-r requirements.txt (line 3)) (3.2)
Requirement already satisfied: urllib3<1.27,>=1.21.1 in /usr/lib/python3.9/site-packages (from requests==2.25.1->-r requirements.txt (line 3)) (1.26.5)
Requirement already satisfied: certifi>=2017.4.17 in /usr/lib/python3.9/site-packages (from requests==2.25.1->-r requirements.txt (line 3)) (2020.12.5)
Collecting pyaes
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/44/66/2c17bae31c906613795711fc78045c285048168919ace2220daa372c7d72/pyaes-1.6.1.tar.gz (28 kB)
Collecting rsa
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/e9/93/0c0f002031f18b53af7a6166103c02b9c0667be528944137cc954ec921b3/rsa-4.7.2-py3-none-any.whl (34 kB)
Collecting wcwidth
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/59/7c/e39aca596badaf1b78e8f547c807b04dae603a433d3e7a7e04d67f2ef3e5/wcwidth-0.2.5-py2.py3-none-any.whl (30 kB)
Collecting pyasn1>=0.1.3
  Downloading https://pypi.tuna.tsinghua.edu.cn/packages/62/1e/a94a8d635fa3ce4cfc7f506003548d0a2447ae76fd5ca53932970fe3053f/pyasn1-0.4.8-py2.py3-none-any.whl (77 kB)
Using legacy 'setup.py install' for Pillow, since package 'wheel' is not installed.
Using legacy 'setup.py install' for pyaes, since package 'wheel' is not installed.
Installing collected packages: pyasn1, wcwidth, rsa, pyaes, Telethon, qrcode, python-socks, prettytable, Pillow, async-timeout
    Running setup.py install for pyaes: started
    Running setup.py install for pyaes: finished with status 'done'
    Running setup.py install for Pillow: started
    Running setup.py install for Pillow: finished with status 'done'
Successfully installed Pillow-8.1.2 Telethon-1.21.1 async-timeout-3.0.1 prettytable-2.4.0 pyaes-1.6.1 pyasn1-0.4.8 python-socks-1.2.4 qrcode-6.1 rsa-4.7.2 wcwidth-0.2.5

python3依赖安装成功...

4、启动bot程序...

bot启动成功...

## 执行结束... 2021-11-13 03:59:29  耗时 46 秒

### CODE REVIEW

**Comment by whyour** [comment]
这个是为了检测依赖如果已安装，不会再次安装，判断条件有点问题

**Comment by fzls** [comment]
> 这个是为了检测依赖如果已安装，不会再次安装，判断条件有点问题

OK，不过pip3 install好像本身如果已安装的话，就会直接跳过的，我觉得好像没必要去判断是否安装过

**Comment by whyour** [comment]
> > 这个是为了检测依赖如果已安装，不会再次安装，判断条件有点问题
> 
> OK，不过pip3 install好像本身如果已安装的话，就会直接跳过的，我觉得好像没必要去判断是否安装过

github的仓储打包是不是有频率限制，workflow会卡在github的打包镜像，无法跳出
